Source/WebKit/NetworkProcess/NetworkProcess.h

```diff
@@ -127,7 +127,7 @@ class Cache;
 enum class CacheOption : uint8_t;
 }
 
-class NetworkProcess : public AuxiliaryProcess, private DownloadManager::Client, public ThreadSafeRefCounted<NetworkProcess>
+class NetworkProcess : public AuxiliaryProcess, private DownloadManager::Client, public ThreadSafeRefCounted<NetworkProcess>, public CanMakeCheckedPtr
```

Comment:
Would be good to include <wtf/CheckedRef.h> since I believe this is where `CanMakeCheckedPtr` is declared.